internal/server/proto/server.proto

```diff
@@ -2924,20 +2925,29 @@ message StatusReport {
 
     // a simple human readable message detailing the Health state
     string health_message = 12;
+
+    // the state of overall health of a deployed application
+    enum Health {
+      UNKNOWN = 0;
+      ALIVE = 1;
+      READY = 2;
+      DOWN = 3;
+      MISSING = 5; // We're expecting it to exist, but it does not.
+      PARTIAL = 4; // Some resources in deployment are OK, others are not OK
+    }
```

Comment:
This was just an oversight from a previous PR - we need this to match the sdk proto definition (https://github.com/hashicorp/waypoint-plugin-sdk/blob/main/proto/plugin.proto#L404)